src/components/TasksTable/index.jsx

```diff
@@ -18,19 +24,39 @@ const sorted = pipe(
 );
 
 @withRouter
-@withStyles(() => ({
+@withStyles(theme => ({
   summary: {
     whiteSpace: 'nowrap',
+    display: 'inline-block',
+    verticalAlign: 'middle',
+    paddingTop: 0,
+    paddingBottom: 0,
   },
   tableWrapper: {
     overflowX: 'auto',
   },
-  link: {
-    textDecoration: 'none',
+  clickedChip: {
+    backgroundColor: theme.palette.secondary.dark,
+    '&:focus': {
+      backgroundColor: theme.palette.secondary.dark,
+    },
   },
   tags: {
     whiteSpace: 'nowrap',
   },
+  summaryItem: {
+    marginLeft: -theme.spacing.unit,
+    padding: theme.spacing.unit,
+  },
+  summaryButton: {
+    '& svg': {
+      transition: transitions.create('fill'),
+      fill: theme.palette.primary.main,
+    },
+    '&:hover svg': {
+      fill: 'white',
+    },
+  },
```

Comment:
This is good to have when using something like a dark theme. Having a gray background with white text makes it hard to see so let's keep the default behavior. Remove `summaryButton`.